examples/static_files/src/tests.rs

```diff
@@ -0,0 +1,71 @@
+use std::fs::File;
+use std::io::Read;
+use rocket::testing::MockRequest;
+use rocket::http::Method::*;
+use rocket::http::Status;
+use rocket::response::Body;
+
+use super::rocket;
+
+fn body_to_bytes(body: Body<&mut Read>) -> Option<Vec<u8>> {
```

Comment:
When I change to the above code snippets, the compiler give the following error.

```
error: no method named `read_to_end` found for type `rocket::response::Body<&mut std::io::Read>` in the current scope
  --> src/tests.rs:13:10
   |
13 |     body.read_to_end(&mut data);
   |          ^^^^^^^^^^^

```
Based on the code
- https://github.com/SergioBenitez/Rocket/blob/master/lib/src/response/response.rs#L13 
- https://github.com/SergioBenitez/Rocket/blob/master/lib/src/response/response.rs#L527

```Body``` type is a enum with generic T where T implement the ```std::io::Read``` trait.
And inside ```Response```, the T type is a ```Box<std::io::Read>``` which is a trait object.

The question is, is there any better way to get the content of enum without using match.